prime-angular-frontend/src/app/modules/site-registration/pages/business-licence-renewal-page/business-licence-renewal-page.component.html

```diff
@@ -52,8 +52,8 @@
 
   <app-page-footer [isInitialEnrolment]="true"
                    [hasSecondaryAction]="true"
-                   primaryActionLabel="Save and Continue"
                    (save)="onSubmit()"
+                   (continue)="onBack()"
```

Comment:
Is it supposed to always go back on both Back and Continue?

Reply:
Yes